frontend/src/citizen-frontend/children/ChildHeader.tsx

```diff
@@ -55,6 +55,8 @@ export default React.memo(function ChildHeader({
           {`${firstName} ${lastName}`}
         </H1>
         {group && <Title>{group.name}</Title>}
+        <br />
+        {unit && <span>{unit.name}</span>}
```

Comment:
This will affect all users and all municipalities, not just duplicates. I think it's OK because the group name is shown anyways, but maybe we should check with if the UX people want to say something about how it looks.